### ISSUE TITLE
Opening modal fires the loader function

### ISSUE THREAD

**Issue by milehighjustin** [Shopify/shopify-app-bridge#338]
## Describe the bug

This may be by design, but every time I open a modal, the remix loader function fires. Is there any way to prevent this? The Polaris modal doesn't do this but both the react and web component ones on app bridge do. 

### To Reproduce

Steps to reproduce the behaviour:

Open a modal. 

### Expected behaviour

The loader to not fire or to be able to prevent it from firing.

### Packages and versions

List the relevant packages you’re using, and their versions. For example:

    "@shopify/app-bridge-react": "^4.1.3",
    "@shopify/cli": "3.58.2",
    "@shopify/polaris": "^12.0.0",
    "@shopify/shopify-api": "^9.5.1",
    "@shopify/shopify-app-remix": "^2.5.0",

## Platform

- OS: Windows and Linux Production
- OS Version: 11
- App: Embedded Shopify Admin

### Additional context

Add any other context about the problem here, such as your app’s configuration:
Remix

**Comment by darrynten** [comment]
@bakura10 I think this is about using the "src" attribute.

You can see #314 to track the issue around static content that is already known.

**Comment by bakura10** [comment]
You're correct @darrynten , my bad. My issue is about the static content indeed.
